modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl

```diff
@@ -16,7 +16,8 @@
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/backend_checks_client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/realm_admin_client/secret",
-        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/configuration_properties"
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/configuration_properties",
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/user_admin_client/secret"
```

Comment:
Does the user admin client have an ID too?

Reply:
Not quite sure what ID you mean?

Reply:
Sorry, I was getting confused - clients have an auto-generated UUID ID, but it's the human readable ID (e.g. "tdr-user-admin") that we use to get the token, so we don't have to store it in the parameter store.

These are the two IDs I was getting confused: https://github.com/nationalarchives/tdr-configurations/pull/21/files#diff-d9db35e987eaae5bb78b20add6ce93f6R1274